settings:

  imu_topic: "/imu/data" 
  gps_topic: "/sensor_msgs/NavSatFix"
  odom_topic: "/gnss_imu_fgo"

  enable_linkstates: false
  map_frame_id : "map"
  base_frame_id: "base_link"
  path_frame_id: "path"
  publish_rate: 20
  pose_topic: "/pose"
  state_topic: "/state"
  


